werkit/aws_lambda/default_handler.py

```diff
@@ -0,0 +1,57 @@
+import os
+import boto3
+import asyncio
+from .parallel import parallel_map_on_lambda
+from botocore.exceptions import ClientError
+
+LAMBDA_WORKER_FUNCTION_NAME = "LAMBDA_WORKER_FUNCTION_NAME"
+env_lambda_worker_function_name = (
+    os.environ[LAMBDA_WORKER_FUNCTION_NAME]
+    if LAMBDA_WORKER_FUNCTION_NAME in os.environ
+    else None
+)
+
+LAMBDA_WORKER_TIMEOUT = "LAMBDA_WORKER_TIMEOUT"
+env_lambda_worker_timeout = (
+    int(os.environ[LAMBDA_WORKER_TIMEOUT])
+    if LAMBDA_WORKER_TIMEOUT in os.environ
+    else None
+)
+
+
+def map_result(result):
```

Comment:
I think `transform_result` would be a clearer name.

Reply:
Fixed in 40c1b1d2df915014df3bbdad532f34540f1c0a14